backend/spec/export_ead_si_overrides_spec.rb

```diff
@@ -0,0 +1,163 @@
+# encoding: utf-8
+require 'nokogiri'
+require 'spec_helper'
+require_relative '../../../../backend/spec/export_spec_helper'
+
+# Used to check that the fields EAD needs resolved are being resolved by the indexer.
+require_relative '../../../../indexer/app/lib/indexer_common_config'
+
+describe 'SI EAD export mappings' do
+
+  #######################################################################
+  # FIXTURES
+  #######################################################################
+
+  def load_export_fixtures
```

Comment:
All these methods in the body of the test file itself is kind of icky and noisy, but this is the state of the [ead export tests in upstream master](https://github.com/archivesspace/archivesspace/blob/master/backend/spec/export_ead_spec.rb), and now is not the time to reinvent the wheel.  Goal here was just some tests proving that our current overrides do work, before we start piling on even more overrides.